front/src/modules/rollingStock/components/RollingStockSelector/WithRollingStockSelector.tsx

```diff
@@ -14,7 +18,9 @@ const WithRollingStockSelector =
     const [rollingStockSelected, setRollingStockSelected] = useState<
       RollingStockWithLiveries | undefined
     >();
-    const rollingStockId = useSelector(getRollingStockID);
+
+    const { getRollingStockID, getRollingStockComfort } = useOsrdConfSelectors();
+    const rollingStockID = useSelector(getRollingStockID);
```

Comment:
Use RollingStockId rather than rollingStockID
```suggestion
    const rollingStockId = useSelector(getRollingStockID);
```

Reply:
Done